src/registrar/models/suborganization.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+from .utility.time_stamped_model import TimeStampedModel
+
+
+class Suborganization(TimeStampedModel):
+    """
+    Suborganization under an organization (portfolio)
+    """
+
+    name = models.CharField(
+        null=True,
```

Comment:
**change before merging** sub org shouldn't be null or blank. Also it's a character field we should have something as a max (max_length)  as a best practice. I would default to a high number, like 1000.